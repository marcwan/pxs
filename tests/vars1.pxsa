        ADDVAR          招呼
        SET             招呼, "你好吗？"
        DEBUGPRINT      招呼
        ADD             招呼, "我想知道！", 招呼
        DEBUGPRINT      招呼
___LABEL:
        EXIT            0