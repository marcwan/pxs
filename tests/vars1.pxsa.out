DEBUGPRINT: 招呼 = 你好吗？
DEBUGPRINT: 招呼 = 你好吗？我想知道！
